docs/postmortems/00-template.md

```diff
@@ -0,0 +1,72 @@
+---
+date: 2019-10-21
+authors: carbon-bot
+---
+
+# Title of postmortem
+
+**Summary**
+
+Ipsum culpa molestias doloribus modi veniam ipsa Voluptatem voluptatum error
```

Comment:
maybe instead of lorem ipsum we can help people fill these sections out by providing questions or comments like in our issue and PR templates

Reply:
On it!